INPUT_SCHEMA.json

```diff
@@ -28,7 +37,13 @@
             "description": "Optionally use a proxy.",
             "default": {},
             "editor": "proxy"
+        },
+        "initialCookies": {
+            "title": "Login Cookies",
+            "type": "array",
+            "description": "Your login cookies will be used to bypass the login wall. See <a href='https://apify.com/vdrmota/twitter-scraper'>ReadMe</a> for instructions.",
+            "editor": "json"
         }
     },
-    "required": ["initialCookies", "handle"]
-}
\ No newline at end of file
+    "required": ["mode", "handle"]
```

Comment:
I would not make "mode" required, I think it could blow out API users. Let's just give it a default value.

Reply:
indeed!